components/camel-pulsar/src/main/java/org/apache/camel/component/pulsar/configuration/PulsarConfiguration.java

```diff
@@ -51,6 +51,8 @@
     private boolean allowManualAcknowledgement;
     @UriParam(label = "consumer", defaultValue = "10000")
     private long ackTimeoutMillis = 10000;
+    @UriParam(label = "consumer", defaultValue = "60000000")
+    private long negativeAckRedeliveryDelayMicros = 60000000;
```

Comment:
Lets use millis for this option as we do that for all others. micros is soo little that the number is hard to understand, so change it to millis thanks